docs/using/event-types.md

```diff
@@ -44,6 +44,83 @@ The custom schema for an event type can be as simple as
 `{ "\additionalProperties\": true }` to allow arbitrary JSON, but will usually
 have a more specific definition :)
 
+### Compatibility modes
+
+Compatibility modes are used to control schema changes. Each mode
+solves a specific problem and thus present different constraints. It
+ranges from being very constraining, allowing just some minor changes
+to allowing everything.
+
+By default, Nakadi uses the forward mode, which is further explained
+bellow, together with the other two modes: compatible and none.
+
+#### Forward
+
+This mode has been designed to allow event type owners to change their
+schemas without breaking existing consumers.
+
+Its called forward because using older schemas it's safe to read
+events generated by newer schemas. On the other hand, it's not
+backward, in the sense that trying to read an older event using a
+newer schema is not safe.
+
+So if the consumer has been developed with schema version `1.5.0` in
+mind, reading events from `1.5.0` on is safe, for example, `1.6.0`. But
+reading event generated by older schemas, like `1.4.0` is not safe.
+
+Supported changes:
+
+1. changes to meta attributes: titles, descriptions and examples.
+2. addition of new attributes.
+3. marking attributes as required.
+
+Consumers reading events from forward compatible event types SHOULD
+ignore event attributes not declared in the event type schema.
+
+This mode is default because it solves the most common requirement:
+guarantee stability of deployed consumer applications when schema
+changes. It's crucial for the robustness of event driven applications.
+
+#### Compatible
+
+This is the most strict mode. It's both forward compatible and
+backward compatible. In practical terms:
+
+1. Consuemrs using older schemas can safely read events validated by
+   newer schemas.
+2. Clients using newer schema can safely read events validated by
+   older schemas.
+
+It's useful for long term data storage and analysis. It's guarantees
+the highest quality data for business intelegence.
+
+Only a subset of json schema is allowed to make sure no accidental
+incompatibilities or ambiguities will be introduced. This mode is
+required to have data stored and processed by the data lake.
```

Comment:
`to have data` -> `for data to be`
OR
change the whole line to `required for the data lake to store and process the data.`